### PR TITLE
Styling current list of repo

### PR DESCRIPTION
**Pull request by chaturangalagama**
Styling current list of repo

### CODE REVIEW

File: views/repo_list.pug [hunk 1]
```diff
@@ -1,9 +1,30 @@
+
 h1
   Github Repositories
 
 .git-repo-areas
 
   .git-repo( ng-repeat="repo in globalState.user.repos()" ng-controller="ListItemCtrl" )
+    - var btnType = '{{repo.name}}'
```
Comment (badoet) [inline views/repo_list.pug:8]: why not use angular to render the background color using `ng-style` instead?

File: views/repo_list.pug [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
     .repo-name
-      span( ng-bind="repo.full_name" )
-      a.link-out( ng-href="{{repo.html_url}}" target="_blank" )
+      a( ng-href="{{repo.html_url}}" ng-bind="repo.name" target="_blank" )
+    br
+    .repo-descript
+      span( ng-bind="repo.description" )
+      br
```
Comment (badoet) [inline views/repo_list.pug:14]: why not using `padding-top` or `padding-bottom` instead of `br`?

Comment (chaturangalagama) [inline views/repo_list.pug]: yes, I could have used that..

File: views/repo_list.pug (hunk 1 above)
Comment (chaturangalagama) [inline views/repo_list.pug]: yes but I thought this way is easier than that..

**Pull request closed without merging**